docker-app/Dockerfile

```diff
@@ -12,14 +12,20 @@ ARG PIP_DISABLE_PIP_VERSION_CHECK=1
 ARG PIP_NO_CACHE_DIR=1
 
 # install psycopg2 requirements
-RUN apt-get update && apt-get install -y libpq-dev python3-dev gcc
+RUN apt update \
```

Comment:
Why apt over apt-get? Looking at best practices article, they always use `apt-get`. https://docs.docker.com/build/building/best-practices/

Also, dockerfile linter (which was used years ago to cleanup the dockefiles and we can reintroduce as part of the CI) has a warning about it: https://github.com/hadolint/hadolint/wiki/DL3027

The warning actually matches my understading about apt vs apt-get.

Reply:
You are completely right. apt is newer (what was the reason for me to use it) but apt-get is recommended.